src/test/service/poap/StartPOAP.test.ts

```diff
@@ -118,12 +121,10 @@ describe('Start POAP', () => {
 	it('should throw ValidationError if user has active event', async () => {
 		// Populate database with user that has active event
 		await poapSettingsDB.insertOne({
-			discordUserId: '80002',
+			discordUserId: '930362313029460717',
```

Comment:
I hope this isn't a real discordUserId from BanklessDAO

Reply:
Here's hoping 🤞😂